### PR TITLE
Prepare September package releases

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: eng/Packages.Data.props [hunk 1]
```diff
@@ -84,9 +84,9 @@
     <PackageReference Update="Azure.Messaging.ServiceBus" Version="7.2.1" />
     <PackageReference Update="Azure.Messaging.WebPubSub" Version="1.0.0-beta.2" />
     <PackageReference Update="Azure.Identity" Version="1.4.0" />
-    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.0.2" />
-    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.0.2" />
-    <PackageReference Update="Azure.Security.KeyVault.Certificates" Version="4.0.2" />
+    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.2.0" />
```
Comment (JoshLove-msft) [inline eng/Packages.Data.props:87]: Why is this being changed?

**Review by JoshLove-msft — APPROVED** [review]

File: eng/Packages.Data.props (hunk 1 above)
Comment (pakrym) [reply eng/Packages.Data.props:87]: Want to pick up a bug fix for the extension package.

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 0.1.0-preview.16 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 0.1.0-preview.16 (2021-09-07)
```
Comment (weshaggard) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:3]: Did we want to change this prerelease label from preview to beta? I know this is a special package that will always remain in preview but was just curious if we thought about updating it.

Comment (pakrym) [reply sdk/core/Azure.Core.Experimental/CHANGELOG.md:3]: Didn't think much about it. We would have to go to 0.2.0-beta.1 but I think that's fine. I'll do it during the next release.

**Pull request merged**